dts/bindings/gpio/ti,tca6424a.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022 Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+ 
+description: TI TCA6424A I2C-based GPIO expander
+ 
+compatible: "ti,tca6424a"
+ 
+include: [gpio-controller.yaml, i2c-device.yaml]
+ 
+properties:
+    label:
+      required: true
+
+    int-gpios:
+      type: phandle-array
+      required: false
+      description: |
+          GPIO connected to the controller INT pin. This pin is active-low and open-drain.
```

Comment:
Please line wrap this to avoid horizontal scrolling in the bindings index.

Reply:
Done.